hugegraph-test/src/main/java/com/baidu/hugegraph/unit/cassandra/CassandraTest.java

```diff
@@ -21,8 +21,9 @@
 
 import java.util.Map;
 
-import org.apache.commons.configuration.Configuration;
-import org.apache.commons.configuration.PropertiesConfiguration;
+import com.baidu.hugegraph.config.OptionSpace;
```

Comment:
the order?

Reply:
after renaming to apache package and enable style-check, then fix the import style uniformly.

Reply:
make sense